### PR TITLE
BrainGlobe-brainglobe-registration (Saarah Hussain)

### PR DESCRIPTION
**Pull request by saarah815**
Proposal for GSoc 2025

### CODE REVIEW

File: GSoC-2025/BrainGlobe - brainglobe-registration (Saarah Hussain).md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  |------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
+  | Week 0  (W/B Jun 2)          | Community bonding: explore codebases, review literature, align project scope with mentors. |
+  | Weeks 1-2 (Jun 9 - Jun 20)   | Implement and refine baseline region selection method (e.g., grid search). Integrate into pipeline with test coverage and toggle for manual/auto mode.      |
+  | Weeks 3-4 (Jun 23 - Jul 4)   | Develop and integrate alternative method (Bayesian/ML). Run early performance checks.                                                                                        |
+  | Week 5 (W/B Jul 7)           | Evaluate both methods across datasets (e.g., similarity, accuracy). Compare performance.                                                                                  |
+  | Week 6 (W/B Jul 14)          | Code cleanup, CLI integration, midterm report, and mentor check-in.                                                                                      |
```
Comment (IgorTatarnikov) [inline GSoC-2025/BrainGlobe - brainglobe-registration (Saarah Hussain).md:48]: `brainglobe-reigstration` doesn't have a CLI as of yet, API integration would be more appropriate here

File: GSoC-2025/BrainGlobe - brainglobe-registration (Saarah Hussain).md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+  | Weeks 7-8 (Jul 21 - Aug 1)   | Add tests for edge cases, improve UI/logging. Begin user documentation and usage tutorials.                                                                                       |
+  | Week 9 (W/B Aug 4)           | Benchmark both methods across atlases. Optimise default parameters.                                                                                       |
+  | Week 10 (W/B Aug 25)         | Draft blog post with visuals and results. Gather mentor feedback.                                                                                             |
+  | Weeks 11-12 (Sep 1 - Sep 12) | Finalise docs, blog, and code. Submit report and prepare follow-up PRs/stretch goals.                                                                                            |
+
+Time commitment: 40hr / week, Mon-Fri
```
Comment (IgorTatarnikov) [inline GSoC-2025/BrainGlobe - brainglobe-registration (Saarah Hussain).md:54]: The timeline is over the 350 hour scope of the large project. You could either decrease the weekly time commitment, or decrease the number of weeks.

File: GSoC-2025/BrainGlobe - brainglobe-registration (Saarah Hussain).md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+  | Weeks 3-4 (Jun 23 - Jul 4)   | Develop and integrate alternative method (Bayesian/ML). Run early performance checks.                                                                                        |
+  | Week 5 (W/B Jul 7)           | Evaluate both methods across datasets (e.g., similarity, accuracy). Compare performance.                                                                                  |
+  | Week 6 (W/B Jul 14)          | Code cleanup, CLI integration, midterm report, and mentor check-in.                                                                                      |
+  | Weeks 7-8 (Jul 21 - Aug 1)   | Add tests for edge cases, improve UI/logging. Begin user documentation and usage tutorials.                                                                                       |
+  | Week 9 (W/B Aug 4)           | Benchmark both methods across atlases. Optimise default parameters.                                                                                       |
```
Comment (IgorTatarnikov) [inline GSoC-2025/BrainGlobe - brainglobe-registration (Saarah Hussain).md:50]: The work in week 9 will most likely be done in Week 5 as the schedule now stands.

**Review by IgorTatarnikov — APPROVED** [review]
This looks great! Just a few small comments, but outside of those I think it's ready to submit. Good luck!

**Comment by saarah815** [comment]
> This looks great! Just a few small comments, but outside of those I think it's ready to submit. Good luck!

Thanks so much @IgorTatarnikov - I've updated the time plan accordingly. Really appreciate the feedback!

**Review by adamltyson — APPROVED** [review]
